CMakeLists.txt

```diff
@@ -197,7 +198,10 @@ endif ()
 if (RAJA_ENABLE_HIP)
   set(raja_depends
     ${raja_depends}
-    hip)
+    blt::hip)
+  set(raja_depends
+    ${raja_depends}
+    blt::hip_runtime)
```

Comment:
Why both? @davidbeckingsale isn't the `blt::hip` target a strict superset of the `blt::hip_runtime` target?

Reply:
I initially thought it was, but I'm not sure that's right.

Reply:
I think it should be okay to just use `blt::hip`. I don't think it's a superset. These are real targets now, that really only add flags and libraries to files (e.g. `blt::hip` adds `-x hip`).

Reply:
the warning suppression ifdefs are in our tutorial exercise files. we have a complete implementation version for each one and a stubbed out one that folks work through and fill in. To make them easily diffable, the latter starts with the former with lines removed. So the warning suppression is merely to make a clean build, but with notes in the code that they need to enable code.

Reply:
Sure, but would it also work to add a do-nothing "use" of the variables? I would think that would let those lines be in both and not produce larger diffs and more work in the exercises.

Reply:
I don't think it really matters much. The warnings that these changes suppress have been in the code for several years. I decided to quiet them because they bugged me. I didn't think too hard about it. If you want me to make a specific change, please let me know.